### ISSUE TITLE
[Task] Poll the user base

### ISSUE THREAD

**Issue by Tratcher** [seamonkeysocial/twitarr#10]
I'd be useful to get feedback from the wider user base about what worked well for them, what didn't, and suggested improvements.

**Comment by grundoon** [comment]
Absolutely. Right now there are some 70+ members in the twitarr slack channel, a Feedback Forum archived on the server itself with I think 59 messages in it to be revisited, and THO indicated in the recent post-cruise survey email that they plan to "solicit and encourage Sea Monkeys’ suggestions and input once we begin that process".
